### PR TITLE
Set default passwords to empty

### LINKED ISSUE #20: Feature Request: Replace the hard coded default secrets with autogenerated ones
There is already a level of autogeneration of secrets, but there are still db passwords hardcoded in the configurations.
It seems to be easier to describe how to extract those from the cluster like in the [gitlab manual](https://docs.gitlab.com/charts/installation/deployment.html#initial-login) and have all of those autogenerated

### PR DESCRIPTION
**Pull request by martialblog**
just so that there is no chance that anyone will use these.

Fixes #20 (Technically not, but we can't do anything else here)

### CODE REVIEW

**Comment by jacksgt** [comment]
To be honest, I'm never sure what to do about secrets in Helm charts.
Some charts generate random secrets, e.g. https://github.com/bitnami/charts/blob/master/bitnami/mariadb/templates/secrets.yaml#L19
Some charts use default credentials and expect users to overwrite them, e.g. https://github.com/nextcloud/helm/blob/master/charts/nextcloud/values.yaml#L69
In my opinion, these approaches are suboptimal, so I'm in favor of removing the default credentials.
However, in this case I would actually unset the values (`null`), and use a `{{ requiredValue ... }` in the template to ensure the value is actually set (and not just empty).

**Comment by martialblog** [comment]
TIL about the `required` function. Yeah makes sense.

> The required function allows you to declare a particular values entry as required for template rendering. If the value is empty, the template rendering will fail with a user submitted error message.

https://helm.sh/docs/howto/charts_tips_and_tricks/

**Comment by martialblog** [comment]
Ok I updated the `values.yaml` the passwords are now set to `null`.

If the `externalDatabase.password` is required and not set the user will receive an error. The Bitnami MariaDB Chart will to the same with the passed values.

I extended the PR to also include the Limesurvey Admin password. Either the user provides one extracts the random password from the Secret.

File: limesurvey/README.md [hunk 1]
```diff
@@ -93,6 +93,16 @@ mariadb:
 In both cases the application will automatically be configured to use these credentials.
 Please refer to the [values.yaml](./values.yaml) for all possible configuration values.
 
+## Configuration and installation details
+
+### LimeSurvey Administrator Password
+
+If the initial Admin Password `limesurvey.admin.password` is not provided it will be set to a random string. You extract the password from the Secret:
```
Comment (jacksgt) [inline limesurvey/README.md:100]: You *can* extract

**Review by jacksgt — APPROVED** [review]
Just one minor error. Otherwise good to go.

File: limesurvey/README.md (hunk 1 above)
Comment (martialblog) [reply limesurvey/README.md]: Me fail english? That's unpossible!

Comment (martialblog) [reply limesurvey/README.md]: Done

**Pull request merged**